src/rcms/fm/app/level1/HCALxmlHandler.java

```diff
@@ -223,6 +223,10 @@ public String stripExecXML(String execXMLstring, ParameterSet<FunctionManagerPar
       for (StringT maskedApp: maskedAppArray) {
         //logger.info("[JohnLogVector] " + functionManager.FMname + ": about to start masking " + maskedApp.getString());
         String[] maskedAppParts = maskedApp.getString().split("_");
+        if (maskedAppParts.length != 2 || HCALlevelOneEventHandler.stringToInt(maskedAppParts[0]) || !(HCALlevelOneEventHandler.stringToInt(maskedAppParts[1]))){
```

Comment:
Fine with me. GUI edit of masked apps is extremely rare anyway.
To be followed up#295. @CiaranGodfrey Can you remove these checks in stripExecXML?